api/candidates/models/candidates.py

```diff
@@ -0,0 +1,37 @@
+from mongoengine import (
+    DateTimeField,
+    Document,
+    EmbeddedDocumentField,
+    EmbeddedDocumentListField,
+    ListField,
+    StringField,
+)
+
+from .companies import Company
+from .deputy import DeputyInfo
+from .elections import Election, ElectionsCount
+from .sessions import Sessions
+from .terms import Term
+
+
+class Candidates(Document):
+    civil_name = StringField(required=True)
+    cpf = StringField(unique=True, sparse=True)
+    voter_id = StringField()
```

Comment:
Just asking... Will `id` be an integer? If so, why is it a `StringField`? If not, wouldn't be better to name it with something which makes it clear what its represent? 
If all of that does not make sense, just ignore 😝